script.js

```diff
@@ -0,0 +1,111 @@
+//1
+function getCentury(year) {
+  if (year % 100 === 0) {
+    year++;
+  }
+
+  const century = Math.ceil(year / 100).toString();
+  let ending = 'th';
+
+  if (century.charAt(1) === '1') {
+    ending = 'st';
+  }
+  if (century.charAt(1) === '2') {
+    ending = 'nd';
+  }
+  if (century.charAt(1) === '3') {
+    ending = 'rd';
+  }
+
+  return `${century}${ending}`;
+}
+
+//2
+function stringToNumbers(input) {
+  const vowels = 'aeioyu';
+
+  const transformCharToNumber = (char) => {
+    if (vowels.includes(char)) {
+      return '1';
+    }
+    return '0';
+  };
+
+  return input.split('').map(transformCharToNumber).join('');
+}
+
+//3
+function spinWords(input) {
+  const reverseWord = (word) => {
+    if (word.length > 5) {
+      return word.split('').reverse().join('');
+    }
+    return word;
+  };
+
+  return input.trim().split(' ').map(reverseWord).join(' ');
+}
+
+// 4
+function highAndLow(str) {
+  const arr = str.split(' ');
+
+  const min = arr.reduce((prev, current) => {
+    return Math.min(prev, current);
+  });
+
+  const max = arr.reduce((prev, current) => {
+    return Math.max(prev, current);
+  });
+
+  return `${min} ${max}`;
+}
+
+// 5
+function createPhoneNumber(arr) {
+  return arr.reduce((acc, currentValue, index) => {
```

Comment:
You need to split the string

```suggestion
  return arr.split('').reduce((acc, currentValue, index) => {
```